Items.cs

```diff
@@ -1,418 +1,430 @@
-        const int Romani_Mask = 78;
-        const int Circus_Leader = 79;
-        const int Kafei_Mask = 80;
-        const int Couple_Mask = 81;
-        const int Truth_Mask = 82;
-        const int Kamaro_Mask = 83;
-        const int Gibdo_Mask = 84;
-        const int Garo_Mask = 85;
-        const int Captain_Hat = 86;
-        const int Giant_Mask = 87;
-        const int Goron_Mask = 88;
-        const int Zora_Mask = 89;
+        public const int MaskPostmanHat = 68;
+        public const int MaskAllNight = 69;
+        public const int MaskBlast = 70;
+        public const int MaskStone = 71;
+        public const int MaskGreatFairy = 72;
+        public const int MaskKeaton = 73;
+        public const int MaskBremen = 74;
+        public const int MaskBunnyHood = 75;
+        public const int MaskDonGero = 76;
+        public const int MaskScents = 77;
+        public const int MaskRomani = 78;
+        public const int MaskCircusLeader = 79;
+        public const int MaskKafei = 80;
+        public const int MaskCouple = 81;
+        public const int MaskTruth = 82;
+        public const int MaskKamaro = 83;
+        public const int MaskGibdo = 84;
+        public const int MaskGaro = 85;
+        public const int MaskCaptainHat = 86;
+        public const int MaskGiant = 87;
+        public const int MaskGoron = 88;
+        public const int MaskZora = 89;
+
         //song
-        const int Song_Soaring = 90;
-        const int Song_Epona = 91;
-        const int Song_Storms = 92;
-        const int Song_Sonata = 93;
-        const int Song_Lullaby = 94;
-        const int Song_NWBN = 95;
-        const int Song_Elegy = 96;
-        const int Song_Oath = 97;
+        public const int SongSoaring = 90;
+        public const int SongEpona = 91;
+        public const int SongStorms = 92;
+        public const int SongSonata = 93;
+        public const int SongLullaby = 94;
+        public const int SongNewWaveBossaNova = 95;
+        public const int SongElegy = 96;
+        public const int SongOath = 97;
+
         //areas/other
-        const int SOUTH_ACCESS = 98;
-        const int WF_ACCESS = 99;
-        const int WF_CLEAR = 100;
-        const int NORTH_ACCESS = 101;
-        const int SH_ACCESS = 102;
-        const int SH_CLEAR = 103;
-        const int EPONA = 104;
-        const int WEST_ACCESS = 105;
-        const int PF_ACCESS = 106;
-        const int GB_ACCESS = 107;
-        const int GB_CLEAR = 108;
-        const int EAST_ACCESS = 109;
-        const int IC_ACCESS = 110;
-        const int ST_ACCESS = 111;
-        const int IST_ACCESS = 112;
-        const int ST_CLEAR = 113;
-        const int EXPLOSIVE = 114;
-        const int ARROW = 115;
-        const int WF_NEW = 116;
-        const int SH_NEW = 117;
-        const int GB_NEW = 118;
-        const int LA_NEW = 119;
-        const int IST_NEW = 120;
+        public const int AreaSouthAccess = 98;
+        public const int AreaWoodFallAccess = 99;
```

Comment:
Should clarify that WoodFall / Snowhead / GreatBay / StoneTower Access/Clear refers to Temples. So rename to `AreaWoodFallTempleAccess` etc.